README.md

```diff
@@ -41,7 +41,7 @@ The file ReadMe.txt in a build bundle describes how to install the library on th
    16. [Cross Platform Build and Test on Linux for MinGW](#2-16)
    17. [Building with C++11 and other compilers](#2-17)
    18. [Static and Shared Libraries](#2-18)
-   19. [Support for bmff files (CR3, HIF, AVIF and HEIC)](#2-19)
+   19. [Support for bmff files (CR3, HEIF and AVIF)](#2-19)
```

Comment:
Hm, I didn't mean remove HEIC (and from main page neither below)... As per [Wikipedia](https://en.wikipedia.org/wiki/High_Efficiency_Image_File_Format), I believe we should have HEIF, HEIC, and AVIF documented.

The HIF/HEIF is the same as JPG/JPEG IMHO, no need to handle specially.

Reply:
@clanmills Maybe I just do a PR with what I had in mind?

Reply:
Yes.  My telepathy is down at the moment!  Can you wait until this PR has been merged into 0.27-maintenance and then open a new PR.

Reply:
Sorry, saw this late and already pushed - just minor formatting changes though...

Reply:
Milos.  Please don't make cosmetic changes to existing code that isn't part of the PR.  It introduces changes between old-master and 0.27-maintenance which makes porting fixes from old-master more tricky.  @neheb did a lot of work last Fall on 'master' and it would be good to see if we can merge his great work into 0.27-maintenance.  

@nehab: I'm sorry to let you know that the Team has decided to hibernate 'master'.  https://github.com/Exiv2/exiv2/issues/1466#issuecomment-787129589

@piponazo  and @hassec have both proposed reformatting all the code with clang-tidy.  I think we should consider doing that on 'main' when that ships as 0.28.  Currently scheduled for 2021-09-15.

Reply:
Apologies Robin, I reverted my commits now.

Reply:
@kmilos Thank you for both making the change and reverting the change. I know you intended well with those changes.